base/cfk-components/cp-version/control-centre.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: platform.confluent.io/v1beta1
+kind: ControlCenter
+metadata:
+  name: controlcenter
+spec:
+  image:
+    application: confluentinc/cp-enterprise-control-center:7.2.0
```

Comment:
7.2.1 is not available, can update these?

Reply:
done